keyboards/1upkeyboards/sweet16/keymaps/ridingintraffic/keymap.c

```diff
@@ -0,0 +1,220 @@
+/*
+tap danc eis turned on in the rules now...
+*/
+#include QMK_KEYBOARD_H
+
+#define _EMOJI 0
+#define _TAPLAND 1
+#define _LEDCNTL 2
+
+
+#define TAPPING_TOGGLE 2
+
+enum custom_keycodes {
+    SHRUG,
+    DISFACE,
+    FU,
+    TFLIP,
+    TFLIP2,
+    SAD_EYES,
+    TPUT,
+    HAPPYFACE,
+    HEARTFACE,
+    CLOUD,
+    CHANFACE,
+    CMDCLEAR
+};
+//Tap Dance Declarations
+enum {
+  TD_EXAMPLE1 = 0,
+  TD_EXAMPLE2,
+  TD_EXAMPLE3,
+  TD_EXAMPLE4
+};
+
+void tap(uint16_t keycode){
```

Comment:
You don't need this anymore, `tap_code` was recently added, and performs this. 

You can continue to use this, if you want, though.